DefInjected/ThingDef/Buildings_Security.xml

```diff
@@ -33,10 +33,10 @@
 	<TrapDeadfall.label>ловушка</TrapDeadfall.label>
 	<TrapDeadfall.description>Простая, но эффективная ловушка.</TrapDeadfall.description>
 
-	<TrapIEDBomb.label>взрывное СВУ</TrapIEDBomb.label>
-	<TrapIEDBomb.description>Самодельное взрывное устройство — артиллерийский снаряд на импровизированной растяжке.</TrapIEDBomb.description>
+	<TrapIEDBomb.label>осколочное СВУ</TrapIEDBomb.label>
+	<TrapIEDBomb.description>Самодельное взрывное устройство — артиллерийский снаряд, присоединённый к импровизированному спусковому механизму. Когда кто-нибудь активирует ловушку, снаряд взоровется. Мина сдетонирует после случайной задержки, что может позволить убежать проворному противнику.</TrapIEDBomb.description>
 
 	<TrapIEDIncendiary.label>зажигательное СВУ</TrapIEDIncendiary.label>
-	<TrapIEDIncendiary.description>Самодельное взрывное устройство зажигательного типа — артиллерийский снаряд на импровизированной растяжке.</TrapIEDIncendiary.description>
+	<TrapIEDIncendiary.description>Самодельное взрывное устройство зажигательного типа — взрывчатка, смешанная с легковоспламеняемым веществом. Имеет простой и надежный спусковой механизм в виде распылителя.</TrapIEDIncendiary.description>
```

Comment:
> Это интересно. А можно почитать про это?

[Тут](https://ludeon.com/forums/index.php?topic=2933.0):

> The en-US language folder lacks one of these because all the original def data is in English so no English data needs to be injected. However, you can see examples from other languages like German

> Ну и что? Это не делает его лучше или авторитетнее. Почему ты уверен, что их текст не взят с потолка?

Французский и даже украинский тексты тоже, по твоему, взяты с потолка